contracts/Bridge.sol

```diff
@@ -30,12 +30,11 @@ contract Bridge is Adminable {
     bytes32[] children; // unordered list of children below this node
   }
 
-  bytes32 public constant GENESIS = 0x4920616d207665727920616e6772792c20627574206974207761732066756e21;
+  bytes32 constant GENESIS = 0x4920616d207665727920616e6772792c20627574206974207761732066756e21;
```

Comment:
Will break node.

Reply:
i think the node has it hardcoded here: https://github.com/leapdao/leap-node/blob/master/src/utils/constants.js#L8